__tests__/app.test.js

```diff
@@ -0,0 +1,23 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const endpoints = require("../endpoints.json");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET: 200 - an array of topic objects, each of which should have the properties 'slug' and 'description'", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((topics) => {
```

Comment:
If the array is empty, the tests won’t be reached, therefore they pass because Jest passes describe blocks that don’t contain tests. Make sure to check the length of the array before asserting inside a forEach 👀